### PR TITLE
Allow to open links in new tab

### LINKED ISSUE #1075: Actions of type WEB can not be opened in a new tab in the browser
### Steps to reproduce
1. Have talk installed
2. Be mentioned in another room
3. Try to right click + open in new tab "View chat"

### Expected behaviour
Should work

### Actual behaviour
It's a button, not a link so doesn't work.

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #1075

### CODE REVIEW

File: src/Components/Action.vue [hunk 1]
```diff
@@ -38,18 +45,22 @@ export default {
 		},
 	},
 
+	computed: {
+		isWebLink() {
+			return this.typeWithDefault === 'WEB'
```
Comment (blizzz) [inline src/Components/Action.vue:50]: where does `typeWithDefault` come from?

Comment (nickvergessen) [reply src/Components/Action.vue:50]: The computed below 🙈

Comment (blizzz) [reply src/Components/Action.vue:50]: I should visit the oculist 😵

**Review by blizzz — APPROVED** [review]

**Pull request merged**